Makefile

```diff
@@ -85,10 +85,21 @@ acceptance: ## Run acceptance tests
 
 # Add @focus above the feature you're hacking on to use this
 .PHONY: focus-acceptance
-focus-acceptance: ## Run acceptance tests with @focus tag
+focus-acceptance:
 	@$(MAKE) build
 	@cd acceptance && go test -tags=acceptance . -args -tags=@focus
 
+feature-%: ## Test a single feature, e.g. make feature-validate_image
```

Comment:
Missing `.PHONY`?

Reply:
I wasn't sure if it would work with %, but perhaps it does.

Reply:
Afact `.PHONY: feature-%` doesn't do anything useful.